tests/Components/test_AttachedComponent.py

```diff
@@ -25,3 +39,14 @@ def test_instantiation(attached_component):
 def test_states(attached_component):
     attached_component.actuation_angle = math.radians(3)
     assert(attached_component.actuation_angle == math.radians(3))
+
+
+def test_position(freebody_component, attached_component):
+    attached_component.set_parent(freebody_component)
+    assert(attached_component.position == Vector2(-0.4, 0.1))
+    freebody_component.angle = 10
+    assert(attached_component.position == Vector2(
+        0.390030722719518, 0.13370129144810267))
+    freebody_component.angle = -10
+    assert(attached_component.position == Vector2(
+        0.281226500541644, -0.30151559726339316))
```

Comment:
It would be nice to also change the parent's position, as the child position is affected by both the parent's position and angle.